src/nvim/testdir/test_functions.vim

```diff
@@ -1576,4 +1576,122 @@ func Test_default_arg_value()
   call assert_equal('msg', HasDefault())
 endfunc
 
+func Test_getmousepos()
```

Comment:
This is not from Vim so this will introduce merge conflicts.

Port this to a lua functional test or make it obvious via comments that it is for Neovim only.

Reply:
I've ported the test to Lua (ae97f25e9d10440cb390ce8b9f9cc50adcc71e02). Thanks!